Makefile

```diff
@@ -6,7 +6,7 @@ endif
 
 # spin up the containers
 start:
-	docker compose -f docker-compose.nginx.yml up -d
+	docker compose -f docker-compose.nginx.yml up
```

Comment:
This is a behavior change that might affect others. Consider adding a start-attached like we have in our other projects for the non-daemon mode.